frontend/src/app/modules/queue-dashboard/components/location-info-list/location-info-list.component.html

```diff
@@ -0,0 +1,110 @@
+      <th rowspan="2"></th>
+      <th colspan="2">{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.osmJobs.label" | translate}}</th>
+
+      <th colspan="3" class="new-col-sec">{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.lastHour.label" | translate}}</th>
+      <th colspan="2" class="new-col-sec">{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.nextHour.label" | translate}}</th>
+    </tr>
+    <tr>
+      <th>{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.pending.label" | translate}}</th>
+      <th>{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.running.label" | translate}}</th>
+
+      <th class="new-col-sec">{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.errors.label" | translate}}</th>
+      <th>{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.jobs.label" | translate}}</th>
+      <th>{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.events.label" | translate}}</th>
+
+      <th class="new-col-sec">{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.jobs.label" | translate}}</th>
+      <th>{{"frontend.de.iteratec.osm.queueDashboard.location-info-list.events.label" | translate}}</th>
+    </tr>
+  </thead>
+  <tbody class="datarows">
+
+    <ng-container *ngFor="let location of locationInfo" #full_row detailsVisible="false">
+      <tr class="queueRow">
+        <td>
+          <strong title="{{location.label}} | {{'frontend.de.iteratec.osm.queueDashboard.location-info-list.lastDate.label' | translate}}: {{parseDate(location.lastHealthCheckDate)}}">{{location.id}}</strong>
+        </td>
+        <td>
+          {{location.agents}}
+        </td>
+        <td>
+          {{location.jobs}}
+        </td>
+        <td>
+          <a *ngIf="location.pendingJobs > 0 || location.runningJobs > 0"  class="jobDetail">
+            <span class="arrow fas fa-chevron-down chevron-large"
+                  #arrow_icon
+                  (click)="full_row.detailsVisible = !full_row.detailsVisible"
+                  [class.fa-chevron-down]="!full_row.detailsVisible"
+                  [class.fa-chevron-up]="full_row.detailsVisible">
+            </span>
+          </a>
+        </td>
+        <td>
+          {{location.pendingJobs}}
+        </td>
+        <td>
+          {{location.runningJobs}}
+        </td>
+
+        <td class="new-col-sec">
+          {{location.errorsLastHour}}
+        </td>
+        <td>
+          {{location.jobResultsLastHour}}
+        </td>
+        <td>
+          {{location.eventResultsLastHour}}
+        </td>
+        <td class="new-col-sec">
+          {{location.jobsNextHour}}
+        </td>
+        <td>
+          {{location.eventsNextHour}}
+        </td>
+      </tr>
+
+      <tr class="job-row" #job_row
+          *ngIf="location.pendingJobs > 0 || location.runningJobs > 0"
+          [style.display]="full_row.detailsVisible ? 'table-row' : 'none'">
```

Comment:
Why do you need to use `display: none` instead of adding `full_row.detailsVisible` to the `*ngIf` directive?